app/models/ipeds_ic_ay.rb

```diff
@@ -2,6 +2,8 @@
 class IpedsIcAy < ActiveRecord::Base
   include Loadable, Exportable
 
+  USE_COLUMNS = [:tuition_in_state, :tuition_out_of_state, :books].freeze
```

Comment:
It looks like these constants are sometimes duplicated, in any case, even when they are not. It would be nice if they were stored somewhere more centrally.

Can you create a lib/gibct_constants.rb and then define these there? Then you can invoke them with GIBCT_Constants::SOME_MORE_DESCRIPTIVE_NAME_THAN_USE_COLUMNS

This will have the added benefit of not having this code duplicated.